packages/api-plugin-promotions/src/index.js

```diff
@@ -32,10 +32,11 @@ export default async function register(app) {
       Promotions: {
         name: "Promotions",
         indexes: [
-          [{ shopId: 1, type: 1, enable: 1, startDate: 1, endDate: 1 }, { name: "c2__shopId__type__enable__startDate_endDate" }],
+          [{ shopId: 1, type: 1, enable: 1, startDate: 1, endDate: 1 }, { name: "shopId__type__enable__startDate_endDate" }],
```

Comment:
This was my fault but can help change it to `enabled`.

```suggestion
          [{ shopId: 1, type: 1, enabled: 1, startDate: 1, endDate: 1 }, { name: "shopId__type__enabled__startDate_endDate" }],
```